configuration/pih/statusData/hiv/hivStatuses.yml

```diff
@@ -50,3 +50,9 @@
   statusDataQuery: "lastDispensedMedication.sql"
   valueExpression: "#if(!$arv1DrugName && !$arv2DrugName && !$arv3DrugName) $fn.translate('pihcore.none.label') #end  #if($arv1DrugName) $arv1Name : $arv1DrugName ($arv1DrugQuantity) <br/> #end #if($arv2DrugName) $arv2Name : $arv2DrugName ($arv2DrugQuantity) <br/> #end #if($arv3DrugName) $arv3Name : $arv3DrugName ($arv3DrugQuantity) #end"
   formatExpression: ""
+
+- id: "lastPrescribedMedication"
+  labelCode: "zl.status.regimen.lastPrescribedMedication"
+  statusDataQuery: "lastPrescribedMedication.sql"
```

Comment:
@ljoedavessenn This looks good.  

I would suggest changing the id and filename from "lastPrescribedMedication" to "lastPrescribedHIVMedication".  It's possible you might want to show other meds (for TB, Syphilis, etc) which would be confusing.